R/STMF.R

```diff
@@ -0,0 +1,125 @@
+#' Read Short-Term Mortality Fluctuations data from HMD and construct a `vital` object for use in other functions
+#'
+#' `read_stmf` reads weekly mortality data from the Short-term Mortality Fluctuations (STMF)
+#' series available in the Human Mortality Database (HMD) <https://www.mortality.org/Data/STMF>)
+#' and constructs a `vital` object suitable for use in other functions.
+#'
+#' @param CNTRY Directory abbreviation from the HMD. For instance, Australia = "AUS".
+#' @return `read_stmf_data` returns a `vital` object combining the downloaded data.
+#'
+#' @author Sixian Tang
+#' @examples
+#' \dontrun{
+#' norway <- read_stmf_data(CNTRY = "NOR")
+#' }
+#'
+#'
+#' @export
+read_stmf_data <- function(CNTRY) {

```

Comment:
Use argument names that are consistent with `read_hmd()` to avoid confusing users. In this case, `CNTRY` should be `country`.